src/base/preferences.cpp

```diff
@@ -57,9 +57,11 @@
 #include "logger.h"
 #include "preferences.h"
 
-Preferences* Preferences::m_instance = 0;
+Preferences *Preferences::m_instance = 0;
 
-Preferences::Preferences() {}
+Preferences::Preferences()
```

Comment:
This is unexpected change. Revert it!

Reply:
Damn uncrastify :), rules should be updated :), done